back-end/app.js

```diff
@@ -17,6 +48,39 @@ app.get('/tv_users', (req, res, next) => {
         })
 });
 
+app.post('/login', function(req, res, next) {
+ //  passport.authenticate('local', function(err, user, info) {
+ //    if (err) {
+	// 	return res.status(500).json({error: 'Issue with Passport authentication1'});
+	// }
+ //    if (!user) {
+	// 	return res.status(403).json({error: 'The login information entered is not correct. Please try again'});
+	// }
+ //    req.logIn(user, function(err) {
+ //      if (err) {
+	// 	return res.status(500).json({error: 'Issue with Passport authentication2'});
+	//   }
+	// 	return res.json({success: 'Successfully logged in user'});
+ //    });
+ //  })(req, res, next);
+ //saving for later
+   axios.post(`https://my.api.mockaroo.com/tv_users.json?key=${process.env.API_KEY_MOCKAROO}&__method=POST`, {
```

Comment:
Since this is just a placeholder for the actual login method, I think using POST is fine here?